_rules/link-in-context-descriptive-5effbb.md

```diff
@@ -3,7 +3,7 @@ id: 5effbb
 name: Link in context is descriptive
 rule_type: atomic
 description: |
-  This rule checks that the accessible name of a link together with its context describe its purpose.
+  This rule checks that the accessible name of a link together with its context describes its purpose.
```

Comment:
I think these changes should go into a separate PR.

Reply:
Good point. I'm grouping all changes in the existing rule in #1530 (including changes requested here that make sense to be backported).